docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -1,41 +1,58 @@
-                promise to resolve before invoking the `journey`.
-                Takes one parameter: a `callback` (function).
-                See <<before-after>>.
-`before`::      Runs a provided function before a single `journey` runs.
-                Takes one parameter: a `callback` (function).
-                See <<before-after>>.
-`afterAll`::    Runs a provided function once, after all the `journey` runs have completed.
-                Takes one parameter: a `callback` (function).
-                See <<before-after>>.
-`after`::       Runs a provided function after a single `journey` has completed.
-                Takes one parameter: a `callback` (function).
-                See <<before-after>>.
+// The synthetics agent exposes an API for creating and running tests, including:
+
+// [horizontal]
+// `journey`::     Tests one discrete unit of functionality.
+//                 Takes two parameters: a `name` (string) and a `callback` (function).
+//                 See <<synthetics-create-journey>>.
+// `step`::        Actions within a journey that should be completed in a specific order.
+//                 Takes two parameters: a `name` (string) and a `callback` (function).
+//                 See <<synthetics-create-step>>.
+// `monitor`::     The `monitor.use` method allows users determine a monitor's configuration on a journey-by-journey basis. 
+//                 If you want two journeys to create monitors with different intervals, for example, you should call
+//                 `monitor.use` in each of them and set the `schedule` property.
+//                 [note: Only relevant for the push command to create monitors in Kibana]
+//                 See <<monitor-use>>.
+// `beforeAll`::   Runs a provided function once, before any `journey` runs.
+//                 If the provided function is a promise, the runner will wait for the
+//                 promise to resolve before invoking the `journey`.
+//                 Takes one parameter: a `callback` (function).
+//                 See <<before-after>>.
+// `before`::      Runs a provided function before a single `journey` runs.
+//                 Takes one parameter: a `callback` (function).
+//                 See <<before-after>>.
+// `afterAll`::    Runs a provided function once, after all the `journey` runs have completed.
+//                 Takes one parameter: a `callback` (function).
+//                 See <<before-after>>.
+// `after`::       Runs a provided function after a single `journey` has completed.
+//                 Takes one parameter: a `callback` (function).
+//                 See <<before-after>>.
 
 TIP: {playwright-url}[Playwright] is a browser testing library developed by Microsoft.
 It's reliable and fast and features a modern API that automatically waits for page elements to be ready.
 
+[discrete]
+[[synthetics-initialize]]
+== Initialize a synthetic project
+
+Start by creating a Node.js project.
```

Comment:
Before this, we need to ask users to install the Synthetics agent via `npm install -g @elastic/synthetics` and then scaffolding a new Synthetics project.